lib/ReactViews/StandardUserInterface/processCustomElements.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import {Nav, Menu, ExperimentalMenu} from './customizable/Groups';
+import {Nav, Menu, ExperimentalMenu, Feedback} from './customizable/Groups';
 
 const GROUP_ELEMENT_TO_KEY_MAPPING = {
     'menu': Menu,
     'nav': Nav,
-    'experimentalMenu': ExperimentalMenu
+    'experimentalMenu': ExperimentalMenu,
+    'feedback' : Feedback
```

Comment:
This line is causing a lint error because of an extra space
```
/home/travis/build/TerriaJS/terriajs/lib/ReactViews/StandardUserInterface/processCustomElements.js
  8:5  warning  Extra space after key 'feedback'  key-spacing
```

```suggestion
    'feedback': Feedback
```

Reply:
fixed!